### ISSUE TITLE
Review extensibility of Value Sets

### ISSUE THREAD

**Issue by cbizon** [clingen-data-model/clingen-interpretation#214]
Only a few of our value sets are fixed.  Is that correct?  I am writing some validation for values, but it may not be worth it...

**Comment by larrybabb** [comment]
it really is an exceptional situation when it is fixed. But when it is it is pretty important. I've noticed this in FHIR as well.  Essentially fixed valuesets are usually control data that is vital to the meta data or infrastructure or standard itself. but it is not necessarily always the case.

**Comment by cbizon** [comment]
I agree with that, but I think we might have swung a little too far that direction.  For example:
```
{
  "id": "SEPIO-CG:65110",
  "type": "ValueSet",
  "description": "Denotes whether the collected evidence is sufficient to meet a specified criteria or not",
  "conceptListExtensibility": {
    "id": "SEPIO-CG:02102",
    "label": "extensible"
  },
  "conceptType": "CriterionEvaluation",
  "concept": [
    {
      "id": "SEPIO:0000223",
      "label": "Met"
    },
    {
      "id": "SEPIO:0000224",
      "label": "Not Met"
    }
  ],
```

I'd say that one should be fixed.

**Comment by larrybabb** [comment]
good catch. I trust your judgement. Please fix the ones that you feel should be fixed. In practice we will learn if we need to make things more flexible.  I suppose we should err on the side of fixed if we cannot foresee any reasonable reason to make it extensible.
